Closr/RestaurantFontSpace.swift

```diff
@@ -0,0 +1,270 @@
+    
+    
+    //RestaurantPrice
+    
+    static let restaurantPriceSpace: spaceStruct = {
+        if Device.is_3_5_inches || Device.is_4_inches {
+            return spaceStruct(originX:201,originY:82,sizeWidth:200,sizeHeight:11)
+        }
+        
+        if Device.is_4_7_inches {
+            return spaceStruct(originX:236,originY:85,sizeWidth:220,sizeHeight:12)
+        }
+        
+        return spaceStruct(originX:259,originY:106,sizeWidth:250,sizeHeight:13.5)
+    }()
+    
+    
+    //restaurantCategory
+    
+    static let restaurantCategorySpace: spaceStruct = {
+        if Device.is_3_5_inches || Device.is_4_inches {
+            return spaceStruct(originX:151,originY:100.5,sizeWidth:200,sizeHeight:8.5)
+        }
+        
+        if Device.is_4_7_inches {
+            return spaceStruct(originX:189.5,originY:106,sizeWidth:220,sizeHeight:10)
+        }
+        
+        return spaceStruct(originX:208,originY:129.5,sizeWidth:250,sizeHeight:11)
+    }()
+    
+    
+    
+    //restaurantImageView
+    
+    static let restaurantImageViewSpace: spaceStruct = {
+        if Device.is_3_5_inches || Device.is_4_inches {
+            return spaceStruct(originX:0,originY:0,sizeWidth:149,sizeHeight:109)
+        }
+        
+        if Device.is_4_7_inches {
+            return spaceStruct(originX:0,originY:0,sizeWidth:175,sizeHeight:128)
+        }
+        
+        return spaceStruct(originX:0,originY:0,sizeWidth:192,sizeHeight:141)
+    }()
+    
+    
+    //cellDistance
+    
+    static let cellDistanceSpace: spaceStruct = {
+        if Device.is_3_5_inches || Device.is_4_inches {
+            return spaceStruct(originX:0,originY:109,sizeWidth:320,sizeHeight:3.5)
+        }
+        
+        if Device.is_4_7_inches {
+            return spaceStruct(originX:0,originY:128,sizeWidth:375,sizeHeight:4.5)
+        }
+        
+        return spaceStruct(originX:0,originY:141,sizeWidth:414,sizeHeight:5)
+    }()
+    
+    
+    //promotionRateView
+    
+    static let promotionRateViewSpace: spaceStruct = {
+        if Device.is_3_5_inches || Device.is_4_inches {
+            return spaceStruct(originX:25,originY:70,sizeWidth:124,sizeHeight:28)
+        }
+        
+        if Device.is_4_7_inches {
+            return spaceStruct(originX:29,originY:81,sizeWidth:146,sizeHeight:33)
+        }
+        
+        return spaceStruct(originX:31,originY:88,sizeWidth:161,sizeHeight:37)
+    }()
+    
+    
+    
+    
+    //addGroupButton
+    
+    static let addGroupButtonSpace: spaceStruct = {
+        if Device.is_3_5_inches || Device.is_4_inches {
+            return spaceStruct(originX:282,originY:82,sizeWidth:25,sizeHeight:25)
+        }
+        
+        if Device.is_4_7_inches {
+            return spaceStruct(originX:330,originY:83,sizeWidth:35,sizeHeight:35)
+        }
+        
+        return spaceStruct(originX:350,originY:95,sizeWidth:45,sizeHeight:45)
+    }()
+    
+    
+    static let imageLabelSpace:CGFloat={
```

Comment:
in front end world, we say Padding rather than Space, for instance imageLabelPadding